libc/test.yaml

```diff
@@ -76,13 +76,18 @@ test:
     - name: unix-socket
       execute: test-libc-unix-socket
       targets:
-        exclude: [armv7a9-zynq7000-qemu]
+        exclude: [armv7a9-zynq7000-qemu, aarch64a53-zynqmp-qemu]
         include: [host-generic-pc]
 
     - name: unix-socket
       execute: test-libc-unix-socket --extra-poll-delay-ms 150
       targets:
         value: [armv7a9-zynq7000-qemu]
+    
+    - name: unix-socket
+      execute: test-libc-unix-socket --extra-poll-delay-ms 150 --transfer_loop_cnt 10
```

Comment:
As you mentioned there is a significant change in execution times between `aarch64a53-zynqmp-qemu` and `armv7a9-zynq7000-qemu` so we should raise the issue about it.

Providing you add the link to the issue here it will be reasonable to use custom argument. However I see that that this test is still failing in CI - please resolve it + raise the issue and then we can merge those changes